### PR TITLE
A couple new posters I made

### PR DESCRIPTION
**Pull request by LevitatingTree**
## About the PR

Added several posters. This is my first PR, so _please_ bare with me.

## Technical details

Basic YAML and JSON stuff

## Media

Image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Added several new posters!

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/Entities/Markers/Spawners/Random/posters.yml [hunk 1]
```diff
@@ -0,0 +1,171 @@
+- type: entity
+  parent: MarkerBase
+  id: RandomPosterAny
```
Comment (Piras314) [inline Resources/Prototypes/_Goobstation/Entities/Markers/Spawners/Random/posters.yml:3]: delete these components that already exist in the non goob files, you can modify them to add your posters, don't re-do them

**Comment by deltanedas** [comment]
bro opened the pr twice to snipe 1000

File: Resources/Prototypes/_Goobstation/Entities/Markers/Spawners/Random/posters.yml (hunk 1 above)
Comment (LevitatingTree) [reply Resources/Prototypes/_Goobstation/Entities/Markers/Spawners/Random/posters.yml]: like so?

**Comment by LevitatingTree** [comment]
i cannot for the life of me figure out why the test is failing

**Comment by LevitatingTree** [comment]
why in tarnation is test failing

**Comment by LevitatingTree** [comment]
[@coderabbitAI](https://github.com/apps/coderabbitai)
test

**Comment by whateverusername0** [comment]
sniped #1000 AND has failed checks
what a noob

**Comment by LevitatingTree** [comment]
> sniped #1000 AND has failed checks what a noob

skill issue tbh

**Comment by LevitatingTree** [comment]
heisentest according to someone smarter than me

**Review by Piras314 — APPROVED** [review]

**Pull request merged**